ComponentKit/Core/Scope/CKComponentScopeHandle.h

```diff
@@ -26,7 +27,7 @@
  handle corresponding to the current scope. Otherwise it returns nil.
  This is only meant to be called when constructing a component and as part of the implementation itself.
  */
-+ (instancetype)handleForComponent:(CKComponent *)component;
++ (instancetype)handleForComponent:(id<CKScopedComponent>)component;
```

Comment:
ahh so this is why the new protocol exists. we can reuse it for… other similar systems.

should we call this `CKScopedObject` instead? `CKScopedElement`? Some other generic name?